tests/plugins/entities/manual/entitiesprocessnumericalforce.md

```diff
@@ -0,0 +1,10 @@
+@bender-tags: bug, 4.20.0, 4941
```

Comment:
I wouldn't use issue tag here. This test is just for sake of coverage entities feature.

Reply:
I think that it's correct now because even if it's for coverage purposes, it's still broken while using `force` option, and this PR fixing it. WDYT?